using_images/other_images/jenkins.adoc

```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
 plug-in] that allows Jenkins slaves to be dynamically provisioned on multiple
 container hosts using Kubernetes and {product-title}.
 
-The Jenkins image entrypoint also provides auto-discovery and auto-configuration
-of the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
+To facilitate the using of the Kubernetes plug-in, {product-title} provides three
+additional images for you consumption.
+
+The first is https://github.com/openshift/jenkins/tree/master/slave-base[base image] for Jenkins slaves.
+It pulls in both required tools (headless Java, the Jenkins JNLP client) and generally useful ones
+(including tar, zip, nss among others) as well as establishing the JNLP slave agent as the entrypoint.  And it provides
+Dockerfiles for both Centos and RHEL images.
```

Comment:
"The image also includes the oc client tooling for invoking command line operations from within Jenkins jobs."